apps/chatter/views.py

```diff
@@ -7,25 +7,8 @@
 
 class RoomViewSet(viewsets.ModelViewSet):
     """API endpoint that allows rooms to be viewed or edited."""
+    authentication_classes = (TokenAuthentication,)
```

Comment:
This is redundant, as it's the default: https://github.com/inputlogic/chatter-api/blob/2ee810e5c98019a1828ea3830f1652664f785f31/project/settings.py#L223

Reply:
Ah! Good to know.